spec.html

```diff
@@ -6447,23 +6447,23 @@ <h1>
       <h1>
         CreateListFromArrayLike (
           _obj_: an ECMAScript language value,
-          optional _elementTypes_: a List of names of ECMAScript Language Types,
+          optional _elementTypes_: either ~all~ or ~property-key~,
```

Comment:
```suggestion
          optional _elementTypes_: ~all~ or ~property-key~,
```
(We don't use `either` after a colon except when it's needed to disambiguate.)

Reply:
While we're changing the interface, maybe the `_elementTypes_` parameter should become non-optional.

Reply:
It's omitted in a bunch of places, so it'd make a lot of churn - i'd prefer to have that be a separate PR if it's needed.

Reply:
It's only omitted 3 times in 262. The AO isn't used at all in 402 or HTML or Web IDL, unless I'm missing something.

I agree it's independent of the point of this PR.